.vib/sonarqube/goss/goss.yaml

```
@@ -0,0 +1,26 @@
+file:
+  /bitnami/sonarqube:
+    exists: true
+    filetype: directory
+    mode: "2775"
+    owner: root
+  /opt/bitnami/sonarqube/conf/sonar.properties:
+    exists: true
+    filetype: file
+    contains:
+     - /web.*port.*={{ .Vars.containerPorts.http }}/
+     - /search.*port.*={{ .Vars.containerPorts.elastic }}/
+     - /postgresql.*:{{ .Vars.postgresql.service.ports.postgresql }}/{{ .Vars.postgresql.auth.database }}/
+     - /jdbc.*username.*={{ .Vars.postgresql.auth.username }}/
+     - /jdbc.*password=.*{{ .Vars.postgresql.auth.password }}/
```

Unify RegEx expressions. You shouldn't be checking for specific separators unless necessary. Prefer wildcards to specific symbols.

```suggestion
     - /web.*port.*{{ .Vars.containerPorts.http }}/
     - /search.*port.*{{ .Vars.containerPorts.elastic }}/
     - /postgresql.*{{ .Vars.postgresql.service.ports.postgresql }}.{{ .Vars.postgresql.auth.database }}/
     - /jdbc.*username.*{{ .Vars.postgresql.auth.username }}/
     - /jdbc.*password.*{{ .Vars.postgresql.auth.password }}/
```